app-testsuite/src/run.ts

```diff
@@ -291,6 +295,11 @@ class App {
     await spawn('node', [CLI, 'deploy'], {
       cwd: this.appDir,
       stdio: 'inherit',
+      env: { ...process.env, REACT_APP_RUNNING_ON: 'realm' },
```

Comment:
Doing nothing, because it assumes remote build will behave in a particular way -- one which would surprise me.  I'm not sure why a comment here would stick out (unfortunately).